QuickDeployment/V2/QuickDeploymentSetup.md

```diff
@@ -303,6 +303,10 @@ A version of this readme file can be found in the [[Quick Deployment Desktop Rea
 > The online version is likely more up to date than the ReadMe you will find on the desktop (not including redacted items like credentials).
 > If there are conflicts between the desktop readme and what you see online, prefer the online version.
 
+:memo: **Note**
+
+When running the `Set-QDEnvironment` script ensure you are logged into the QDE VM with the _local admininistrator_ account. This is necessary for SQL to work properly initially.
+
```

Comment:
Not really a fan of stacking successive note blocks one after another... maybe make this a part of the previous one?

```suggestion
> The online version is likely more up to date than the ReadMe you will find on the desktop (not including redacted items like credentials).
> If there are conflicts between the desktop readme and what you see online, prefer the online version.
>
> When running the `Set-QDEnvironment` script ensure you are logged into the QDE VM with the _local admininistrator_ account.
> This is necessary for SQL configuration to work properly initially.

```

Reply:
done!